R/findOptimalThreadCount.R

```diff
@@ -5,6 +5,8 @@
 #' @param rowCount The number of rows in the \code{data.table}.
 #'
 #' @param colCount The number of columns in the \code{data.table}.
+#' 
+#' @param iterations The number of times the benchmarks are to be run.
```

Comment:
do you have a strong reason to use iterations as the name here?
If not, please use times, for consistency.

Reply:
> do you have a strong reason to use iterations as the name here? If not, please use times, for consistency.

Nope, and sure